lib/itamae/resource/remote_directory.rb

```diff
@@ -9,6 +9,8 @@ class RemoteDirectory < Base
       define_attribute :mode, type: String
       define_attribute :owner, type: String
       define_attribute :group, type: String
+      define_attribute :recursive_mode, type: [String, Symbol], default: :true
```

Comment:
I think better way of recursive chown/chmod in default in my local usage.
But, this patch changes behavior from current version.   It is not good.

I change to boolean false in default.